lib/mix/tasks/ash_oban.install.ex

```diff
@@ -99,6 +101,32 @@ if Code.ensure_loaded?(Igniter) do
           end
         end
       )
+      |> Igniter.Project.Config.configure(
+        "config.exs",
+        app_name,
+        [Oban, :plugins],
+        [{cron_plugin, []}],
+        updater: fn list ->
+          case Igniter.Code.List.prepend_new_to_list(list, {cron_plugin, []}, fn old, new ->
+                 if Igniter.Code.Tuple.tuple?(old) do
+                   with {:ok, plugin} <- Igniter.Code.Tuple.tuple_elem(old, 0) do
```

Comment:
The fall through clause here will return `:error` which is always truthy. Since there is only one match, I'd do 

```elixir
case Igniter.Code.Tuple.tuple_elem(old, 0) do
  {:ok, zipper} -> Igniter.Code.Common.nodes_equal?(plugin, cron_plugin)
  :error -> false
end
```